django/db/models/expressions.py

```diff
@@ -620,6 +624,8 @@ def as_sql(self, compiler, connection):
 
 
 class RawSQL(Expression):
+    contributes_to_group_by = True
```

Comment:
Are we missing a regression test for this addition? No tests are failing for me if I remove it (checked SQLite and PostgreSQL).